src/app/components/competitions/create-competition/create-competition.component.scss

```diff
@@ -1,3 +1,10 @@
 .first-field {
   margin-top: 0.3rem;
 }
+
+.lds-dual-ring {
+  display: flex;
```

Comment:
is PrimeNg used in this project? If it is, you can use primeflex classes to simplify scss files